### PR TITLE
Initial draft of the darktable 3.6 blog post

### LINKED ISSUE #140: Initial content for darktable 3.6 release article
As last time, I've created a page on hackmd for us to begin drafting the release article for darktable 3.6.

For now I've assumed we're going for a summer solstice release (20 June 2021).

https://hackmd.io/4Y6kfnGFT9OXSM1wZ7ljMw

### PR DESCRIPTION
**Pull request by paperdigits**
Initial draft of release blog post.

Resolves #140

### CODE REVIEW

File: content/blog/2021-07-03-darktable-3.6/index.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+The darktable default for blending modes is to apply the module output over the module input with some optional transparency (using classical [alpha compositing](https://en.wikipedia.org/wiki/Alpha_compositing) and assuming a fully opaque input). The reversed modes will instead apply the input over the output. 
+
+As of darktable 3.4, the mask feathering (using a guided filter) was always applied before the mask blurring (using a gaussian blur). Both techniques aim at ensuring contiguous image regions are masked continuously, without holes, especially in the presence of noise and when using chromaticity masks. However, the gaussian blur is not edge-aware and can leak out of the boundaries of masked regions, producing edge halos in some situations, whereas the guided filter is taped closely to edges. While the gaussian blur is more effective to deal with noise, using it after the guided filter voids its edge-awareness. Applying in reversed order may help to more aggressively correct noise while still retaining the edge-aware properties of the guided filter (use small blurring radii).
```
Comment (kofa73) [inline content/blog/2021-07-03-darktable-3.6/index.md:107]: I'm not a native speaker, but I have problems with this sentence:

> While the gaussian blur is more effective to deal with noise, using it after the guided filter voids it edge-awareness.

- Shouldn't _Gaussian_ be spelt with a capital letter?
- _it_ and _its_ in the sentence: the edge-aware filter is the guided one, so logic says the _its_ refers to the guided filter; however, the _it_ of _using it_ seems to refer back to the Gaussian.

**Comment by kofa73** [comment]
After running through LanguageTool.org:
- if for 'color' the US spelling is used, 'grey' should be replaced with 'gray' for consistency. Similarly, analyse vs analyze.
- languagetool insists that 'side-effect' should be written as 'side effect'. https://www.merriam-webster.com/dictionary/side%20effect supports this.
- 'pixellate' - while this word is not in MW, https://www.merriam-webster.com/dictionary/pixelated is. Maybe the verb should be spelt with a single 'l', too. Google returns ~52k vs 860k hits for 'pixellate' and 'pixelate', respectively.
- _When shapes are drawn on the image they are actually drawn on the original raw image co-ordinates_ - I think a comma is missing before _they_: _When shapes are drawn on the image, they are actually drawn..._
- outside of -> outside? E.g.  'retouch can use source spots outside --of-- the cropped image area'
- _"dual demosaicing" modes have been added in a similar manner as Rawtherapee_ -> shouldn't that be 'similar to'? And could 'in a manner similar to' be replaced with 'similarly'?
- _This option is only available for RAW files as it depends on details from demosaic stage._ -> comma after 'RAW files'?
-  'adaptative manifolds' -> adaptive manifolds? (23 Google hits vs 4k)
- Furthermore (see next point) this scope does not provide specific guidance to correct for specific image content. -> comma after _Furthermore_ (or after _(see next point)_)?
- These elements default to a 16:9 aspect ratio but can be resized - comma before _but_?
- 'Chromatic Abberations' -> C. Aberrations
- While in folder view you can toggle between - comma after _view_?
- The export module has been added to left-hand panel in the darkroom -> to *the* left-hand panel?

**Comment by paperdigits** [comment]
Can you please make a pull request?

**Comment by jenshannoschwalm** [comment]
Just for the 3.8 outlook, there will be a LMMSE demosaicer, good for high ISO images.

**Comment by kofa73** [comment]
Done. Take it with a huge grain of salt.

**Comment by elstoc** [comment]
Thanks everyone

**Pull request merged**